atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -116,50 +111,53 @@
     final ConcurrentMap<TableReference, Set<Cell>> cellsRead = Maps.newConcurrentMap();
     final ConcurrentMap<TableReference, Set<RowRead>> rowsRead = Maps.newConcurrentMap();
 
-    public SerializableTransaction(MetricsManager metricsManager,
-                                   KeyValueService keyValueService,
-                                   TimelockService timelockService,
-                                   TransactionService transactionService,
-                                   Cleaner cleaner,
-                                   Supplier<Long> startTimeStamp,
-                                   ConflictDetectionManager conflictDetectionManager,
-                                   SweepStrategyManager sweepStrategyManager,
-                                   long immutableTimestamp,
-                                   Optional<LockToken> immutableTsLock,
-                                   PreCommitCondition preCommitCondition,
-                                   AtlasDbConstraintCheckingMode constraintCheckingMode,
-                                   Long transactionTimeoutMillis,
-                                   TransactionReadSentinelBehavior readSentinelBehavior,
-                                   boolean allowHiddenTableAccess,
-                                   TimestampCache timestampCache,
-                                   ExecutorService getRangesExecutor,
-                                   int defaultGetRangesConcurrency,
-                                   MultiTableSweepQueueWriter sweepQueue,
-                                   ExecutorService deleteExecutor,
-                                   boolean validateLocksOnReads,
-                                   Supplier<TransactionConfig> transactionConfig) {
+    public SerializableTransaction(
+            MetricsManager metricsManager,
+            KeyValueService keyValueService,
+            TimelockService timelockService,
+            TransactionService transactionService,
+            Cleaner cleaner,
+            Supplier<Long> startTimeStamp,
+            ConflictDetectionManager conflictDetectionManager,
+            SweepStrategyManager sweepStrategyManager,
+            long immutableTimestamp,
+            Optional<LockToken> immutableTsLock,
+            PreCommitCondition preCommitCondition,
+            AtlasDbConstraintCheckingMode constraintCheckingMode,
+            Long transactionTimeoutMillis,
+            TransactionReadSentinelBehavior readSentinelBehavior,
+            boolean allowHiddenTableAccess,
+            TimestampCache timestampCache,
+            ExecutorService getRangesExecutor,
+            int defaultGetRangesConcurrency,
+            MultiTableSweepQueueWriter sweepQueue,
+            ExecutorService deleteExecutor,
+            boolean validateLocksOnReads,
+            Supplier<TransactionConfig> transactionConfig,
+            TransactionLockWatchingCacheView transactionCache) {
         super(metricsManager,
```

Comment:
If the we want to have this PR have this diff should `metricsManager` also be on a new line.